ui/pages/confirmations/confirm-transaction-base/confirm-transaction-base.container.js

```diff
@@ -129,6 +125,23 @@ function addressIsNew(toAccounts, newAddress) {
   return !foundMatching;
 }
 
+function getTokenToAddress(data, type) {
+  const transactionData = parseStandardTokenTransactionData(data);
+
+  if (
+    [
+      TransactionType.tokenMethodTransferFrom,
+      TransactionType.tokenMethodSafeTransferFrom,
+      TransactionType.tokenMethodTransfer,
+    ].includes(type)
+  ) {
+    const value = transactionData?.args?._to || transactionData?.args?.to;
```

Comment:
Minor, but since this is the core logic of the function, we could also return early if the method does not match so that this is no longer nested.

Reply:
Changed to early return.